test/posix_time/testmicrosec_time_clock.cpp

```diff
@@ -64,6 +64,17 @@ main()
     ptime t2 = microsec_clock::local_time();
     std::cout << t2 << std::endl;
 
+    // sometimes on very slow systems (github actions) a long time 
```

Comment:
Should we just consider removing this test from the test set (can still compile it) since it's always been finicky?  I mean it was one thing when I was developing the library and wanted to be sure this property held -- and it would be something to check on a new platform, but I'm not sure as a recurring test it's worth the trouble.

Reply:
I tested my fix by suspending it at the right moment and it does skip this, I've seen it correct the issue so I would recommend leaving it as is since it passes now.

Reply:
roger that

Reply:
I take it back... look at this

https://ci.appveyor.com/project/jeking3/date-time-1evbf/builds/42545758/job/6o33narnc03htcs2?fullLog=true#L3633

Reply:
@JeffGarland I would consider the change I made an improvement, I would like to merge it anyway (CI is *so* expensive on this!) and then if you want to kill the test feel free.  It does seem to fail on the VS2013/2015 test runner specifically, which is interesting.